safe-global/web-core
# fix: switch import/export icons

## What it solves

Incorrect address book icons

## How this PR fixes it

The previous import and export icons have been switched.

## How to test it

Open the address book tab and observe the icons not correctly match [the design](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=25%3A5752).

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/199431649-a95ac6bd-3dd4-44d2-beb4-22a7380c5b07.png)
